### PR TITLE
🎉 New Source: Demo - Youtube Analytics Business

### PR DESCRIPTION
**Pull request by bazarnov**
## What
Resolving: https://github.com/airbytehq/airbyte/issues/19017

### CODE REVIEW

**Review by grubberr — APPROVED** [review]

**Comment by bazarnov** [comment]
/publish connector=connectors/source-youtube-analytics-business

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml
Comment (timroes) [inline airbyte-config/init/src/main/resources/seed/source_definitions.yaml:1420]: Could we start for new sources to use human readable ids as outlined in https://github.com/airbytehq/airbyte/issues/18447. @sherifnada @evantahler While we might not want to switch all existing because of the migration pain, is there any reason why new connectors couldn't follow a new format, e.g. having an ID of `com.airbyte.youtube-analytics-business` as an ID?

Comment (bazarnov) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:1420]: this new source will exist for only a few days.... can we pass it as is because it will be removed right after the Google approves new scope for Youtube analytics for business.

Comment (timroes) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:1420]: yes sorry I wasn't aware of the demo nature of this when I wrote this. Please feel free to ignore for this connector.

**Comment by girarda** [comment]
@bazarnov is this new source temporary?

An alternative to copying the source would've been to create and publish a docker image with your change and point to it in `source_definitions`

**Comment by timroes** [comment]
@bazarnov Even if you make temporary changes like this, please make sure you wait for the CI run to finish before merging, so we're not running into the risk of breaking master due to some bad commit. (e.g. a single space too much/less in the frontend code would fail our code formatting checks).

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml
Comment (sherifnada) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:1420]: @timroes I believe a large amount of the java codebase would need to change because it expects the UUID format for all definitions. See https://github.com/airbytehq/airbyte/blob/master/airbyte-api/src/main/openapi/config.yaml#L2956

**Pull request merged**
